lib/tasks/yarn-install.js

```diff
@@ -89,7 +89,7 @@ module.exports = function yarnInstall(ui, zipFile) {
 
             const observable = yarn(args, {
                 cwd: ctx.installPath,
-                env: {NODE_ENV: 'production'},
+                env: {NODE_ENV: 'production', YARN_IGNORE_PATH: true},
```

Comment:
minor nitpick - would it make sense to keep all the values here as strings? I know Node internally converts the values to strings, but it may make sense to be explicit here for consistency

```suggestion
                env: {NODE_ENV: 'production', YARN_IGNORE_PATH: 'true'},
```